app/src/controllers/pagination/constants.js

```diff
@@ -1,10 +1,33 @@
+import { NAMESPACE as NAMESPACE_LAUNCH } from '../launch/constants';
+import { NAMESPACE as NAMESPACE_LOG } from '../log/constants';
+
 export const MERGE_PAGINATION = 'pagination/mergePagination';
 export const CHANGE_PAGE = 'pagination/changePage';
 export const CHANGE_SIZE = 'pagination/changeSize';
 export const PAGE_KEY = 'page.page';
 export const SIZE_KEY = 'page.size';
 
+export const DEFAULT_PAGE_SIZE = 250;
+export const MIN_PAGE_SIZE = 1;
+export const MAX_PAGE_SIZE = 300;
+
 export const DEFAULT_PAGINATION = {
   [PAGE_KEY]: 1,
-  [SIZE_KEY]: 50,
+  [SIZE_KEY]: DEFAULT_PAGE_SIZE,
+};
+
+const NAMESPACE_ITEM0 = 'item0';
+const NAMESPACE_ITEM1 = 'item1';
+const NAMESPACE_ITEM2 = 'item2';
+
+const LAUNCH_PAGE_SIZE_STORAGE_KEY = 'launchPageCount';
```

Comment:
All keys can be calculated in runtime using corresponding namespaces

Reply:
Fixed.

Reply:
Fixed.